packages/render-fidelity-tools/src/workflows/test-fidelity.ts

```diff
@@ -44,6 +44,8 @@ try {
 
 let scenarioWhitelist: Set<string>|null = null;
 
+// default update screenshots command takes 4 arguments. If there's more than 4,
+// user has specified scenarios to test
```

Comment:
Is there any command-line help shown for `update-screenshots`? If there is, we should add this information there. Also, is this actually true? Based on your logic below I think the whitelist scenarios and renderers can appear in any order, correct?

Reply:
No there isn't . I just added an explanation in the ReadMe,  hope it makes things more clear.

By 3 arguments (i said 4 before by mistake), i mean the original command has already taken up 3 arguments.  
 "update-screenshots": "node --experimental-modules ./lib/workflows/update-screenshots.js ./test/config.json",
I just want to put a comment here to explain why it's makes a condition to check whether the lengths of arguments are more than 3 or not (i myself was confused with that at first)